styles.css

```diff
@@ -1,826 +1,980 @@
-  height: 48px;
-  padding-left: 5px;
-}
-
-.textarea-container textarea {
-  font-family: Poppins, sans-serif;
-  border: 1px solid transparent;
-  border-radius: 8px;
-  background: #fff;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  width: 80%;
-  margin-top: 20px;
-  height: 170px;
-  padding-left: 5px;
-}
-
-button {
-  width: 140px;
-  height: 48px;
-  margin-left: 40px;
-  border: 1px solid #6070ff;
-  font-family: Poppins, sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  background: #fff;
-  border-radius: 8px;
-  color: #6070ff;
-  align-items: center;
-  text-align: center;
-  letter-spacing: 0.03em;
+/* HTML5 Reset Stylesheets. */
+
+/* -----Header Section ---- */
+@media only screen and (min-width: 376px) {
+  header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 1rem;
+  }
+
+  .logo {
+    margin: 4px 199px 4px 12px;
+  }
+
+  .menu {
+    margin: 4px 24px;
+  }
+
+  nav {
+    display: none;
+  }
+
+  /* -----Header Section ---- */
+
+  .wrapper {
+    position: relative;
+    background: #e5e5e5;
+    height: auto;
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  section.intro {
+    width: 100%;
+    height: auto;
+    background-image: url("/img/header-shapes-mobile.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-color: #fff;
+    padding-bottom: 10px;
+    padding-top: 5px;
+  }
+
+  .title.description {
+    margin: 144px 24px 12px;
+    width: auto;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+  }
+
+  h1.title {
+    margin-bottom: 12px;
+  }
+
+  p.description {
+    margin-bottom: 12px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .social.container {
+    margin: 12px 24px;
+    height: auto;
+  }
+
+  ul.social-icons {
+    display: flex;
+    margin-top: 12px;
+    align-items: center;
+  }
+
+  ul.social-icons li img {
+    margin-right: 19.01px;
+  }
+
+  /* "Works" section */
+
+  .works {
+    display: grid;
+    margin: 114px 24px;
+    grid-template-rows: 220px 80px 76px 24px 72px;
+    row-gap: 12px;
+    background: #fff;
+    margin-bottom: 12rem;
+    justify-items: start;
+    justify-content: center;
+    align-items: center;
+    padding: 16px;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+  }
+
+  .about {
+    display: grid;
+    margin: 114px 24px;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 25px;
+    background: #fff;
+    margin-bottom: 12rem;
+    width: 100%;
+    height: 924px;
+    justify-items: start;
+    justify-content: center;
+    align-items: center;
+    padding: 16px;
+    border-right: 16px;
+  }
+
+  .grid-container-desktop-about {
+    width: 447px;
+    height: 392px;
+    display: grid;
+    grid-template-rows: 1fr 2fr 1fr 1fr;
+    row-gap: 12px;
+    align-items: center;
+    justify-content: center;
+  }
+
```

Comment:
- Please, kindly adjust your style so this part of the about section takes up 60% width so that there will much space for the paragraph to fill and not let the paragraph go beyond 3 or 4 lines below and also reduce the space at the top as its too much compared to the Figma design:


![adjust](https://user-images.githubusercontent.com/59397159/187049838-63679d55-6cba-47f6-942d-0d9d21cd2bde.png)